test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -132,17 +132,12 @@ test
     await setupLoopbackFallback(t);
     const deviceChallengePollPageObject = new DeviceChallengePollPageObject(t);
     await t.expect(deviceChallengePollPageObject.getBeaconClass()).contains(BEACON_CLASS);
-    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Sign in using Okta Verify on this device');
+    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Click "Open Okta Verify" on the browser prompt');
     const content = deviceChallengePollPageObject.getContent();
-    await t.expect(content).contains('Please click');
-    await t.expect(content).contains('Open oktaverify.app');
-    await t.expect(content).contains('if you see the system dialog.');
-    await t.expect(content).contains('If nothing prompts from the browser,');
-    await t.expect(content).contains('click here to launch Okta Verify, or download & run Okta Verify.');
-    await t.expect(content).contains('If nothing prompts from the browser,');
-    await t.expect(content).contains('click here');
-    await t.expect(content).contains('to launch Okta Verify, or');
-    await t.expect(content).contains('download & run Okta Verify.');
+    await t.expect(content).contains('Didn’t get a prompt?');
+    await t.expect(content).contains('Launch Okta Verify');
+    await t.expect(content).contains('Don’t have Okta Verify?');
+    await t.expect(content).contains('Download here');
```

Comment:
In the future, we should try to be more specific with these assertions by continuing to use the `DeviceChallengePollPageObject` element getters. Checking the entire `content` container for fuzzy-text matching doesn't give us any validation that the text appears in the correct place, just that it renders _somewhere_. Mind creating a follow-up JIRA to improve these tests?